interpreter/text/arrange.ml

```diff
@@ -452,17 +452,17 @@ let simd_loadop (op : simd_loadop) =
       ) in
     memop ("load" ^ suffix) op (packed_size sz)
 
-let simd_loadlaneop (op : simd_loadlaneop) =
+let simd_loadlaneop op i =
```

Comment:
Suggestion: factor away the `i` parameter and make this into `simd_laneop`, following AST structure.